draft-ietf-jsonpath-base.md

```diff
@@ -1149,23 +1149,22 @@ Some examples:
 > is a repository for ABNF taken from {{-json}}.  This needs to be
 > deduplicated with definitions above.
 
+<!-- Remnants of the expression language ABNF we need to keep -->
+
 ~~~~ abnf
-number = [ minus ] jsint [ frac ] [ exp ]
-decimal-point = %x2E       ; .
-digit1-9 = %x31-39         ; 1-9
-e = %x65 / %x45            ; e E
-exp = e [ minus / plus ] 1*DIGIT
-frac = decimal-point 1*DIGIT
-jsint = zero / ( digit1-9 *DIGIT )
-minus = %x2D               ; -
-plus = %x2B                ; +
-zero = %x30                ; 0
+number = int [ frac ] [ exp ]
```

Comment:
Parson my ignorance, but where is `int` defined?

Reply:
3.5.4.  Index Selector, second ABNF block

Reply:
Ah, of course. Thanks.